### ISSUE TITLE
Only the author mentions half the bleu-4 score

### ISSUE THREAD

**Issue by CallMeRabbit** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#198]
The model was trained on flicker8k, but the results achieved only half the BLEU-4 score mentioned by the authors (about 0.14-0.15). I have not modified any parameters in train.py. May I ask why such a result occurs?
![image](https://github.com/sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning/assets/96768788/bcc88ff3-cba2-4632-a2d8-ae45b8822ce5)
![屏幕截图 2023-11-24 101006](https://github.com/sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning/assets/96768788/51eb8ad5-bfc4-4ded-b0ed-f5a44902c792)
![屏幕截图 2023-11-24 101110](https://github.com/sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning/assets/96768788/66a24e56-52f9-4bc2-ae03-0cd64569c79b)

**Comment by yiliSept** [comment]
I also have the problem

**Comment by Kevinskt** [comment]
I also have the problem
![{E50AD2CD-CD9E-4cc7-8986-D358CA0B86BD}](https://github.com/sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning/assets/87940475/ad78af41-014b-4e12-94e0-9c29f3845d6b)

**Comment by togawasakikom** [comment]
Yes, I also encountered the same problem. The model did not perform well on the validation machine. I tried using other convolutional networks to extract image features and pre-trained word embedding vectors, but the results were similar and did not improve.
[log0.txt](https://github.com/user-attachments/files/16173439/log0.txt)
